aplib.net-demo/Assets/Scripts/WFC/GridPlacer.cs

```diff
@@ -193,6 +193,7 @@ private void PlaceDoors(int x, int z, Room room)
             {
                 // # Calculate where the door should be placed
 
+                // North is in the negative x direction
```

Comment:
I see that that is how you coded the door placement, but why? I oriented all the prefabs and tile to have North be positive X.

Reply:
Yes, but the grid is generated with north in the negative x direction.